build/custom-ci-Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openshift/origin-release:golang-1.14
+RUN yum -y install epel-release --disablerepo=epel && yum clean all
+RUN yum -y install python3-pip && yum clean all
+RUN pip3 install -U pip
+RUN pip3 install ansible jmespath molecule more-itertools openshift yamllint
```

Comment:
remove these, add `yum -y install make`

Reply:
```
go get -u github.com/onsi/ginkgo/ginkgo
```

Reply:
There may already be a testing base image that we could use for ginkgo tests

Reply:
Good idea Shawn